### PR TITLE
Tolino Page 2 support

### LINKED ISSUE #455: tolino page 2
Coming from #11264, KOReader works just fine with the `android-arm` apk.
The drivers that are working are "Tolino Ntx" and "Tolino Root" for lights and "Freescale/NTX" for e-ink. The driver test also states `Device already supported, EPD: freescale, Lights: tolino`. The only two minor concerns I have are

1. Both working light drivers have a warmth slider but the device doesn't support warmth.
2. KOReader is unable to enable/disable the frontlight, it can only change the brightness. Toggling the light has to be done outside of KOReader by tapping the android status bar.

<details>
<summary><code>test.log</code></summary>

```
Device info:
Manufacturer: rakuten kobo inc.
Brand: rakutenkobo
Model: tolino
Device: ntx_6sl
Product: ntx_6sl
Hardware: e60qv0
Platform: imx6


Test logs:
```

</details>

The device uses Android 4.4.2 with kernel version `3.0.35-gb5779db-dirty`

_Originally posted by @RubixDev in https://github.com/koreader/koreader/issues/8482#issuecomment-1868903843_

### PR DESCRIPTION
**Pull request by RubixDev**
Closes #455

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/android-luajit-launcher/458)

### CODE REVIEW

**Comment by pazos** [comment]
I understand that this fixes your two concerns in #455 and you actually tested it using a self built apk.

Is that ok?

**Comment by RubixDev** [comment]
Yes that is correct. A self-built apk with those changes is currently running on my Tolino Page 2 and is working as expected with both those concerns resolved

File: app/src/main/java/org/koreader/launcher/device/DeviceInfo.kt [hunk 1]
```diff
@@ -139,7 +139,8 @@ object DeviceInfo {
         TOLINO_EPOS,
         TOLINO_SHINE3,
         TOLINO_VISION4,
-        TOLINO_VISION5
+        TOLINO_VISION5,
+        TOLINO_PAGE2
```
Comment (pazos) [inline app/src/main/java/org/koreader/launcher/device/DeviceInfo.kt:143]: sorted, please

File: app/src/main/java/org/koreader/launcher/device/DeviceInfo.kt [hunk 2]
```diff
@@ -232,6 +233,7 @@ object DeviceInfo {
     private val TOLINO_SHINE3: Boolean
     private val TOLINO_VISION4: Boolean
     private val TOLINO_VISION5: Boolean
+    private val TOLINO_PAGE2: Boolean
```
Comment (pazos) [inline app/src/main/java/org/koreader/launcher/device/DeviceInfo.kt:236]: sorted, please

File: app/src/main/java/org/koreader/launcher/device/DeviceInfo.kt [hunk 3]
```diff
@@ -540,6 +542,7 @@ object DeviceInfo {
             && !HARDWARE.contentEquals("e60k00")
             && !HARDWARE.contentEquals("e60q50")
             && !HARDWARE.contentEquals("e70k00")
+            && !HARDWARE.contentEquals("e60qv0")
```
Comment (pazos) [inline app/src/main/java/org/koreader/launcher/device/DeviceInfo.kt:545]: sorted, please

**Comment by RubixDev** [comment]
I sorted it correctly now

**Comment by hugleo** [comment]
> I sorted it correctly now

Remove unused imports:

https://app.codacy.com/gh/koreader/android-luajit-launcher/pullRequest?prid=13309875

**Comment by RubixDev** [comment]
> Remove unused imports

also done

**Review by pazos — APPROVED** [review]
lgtm, thanks!

**Pull request merged**